member.py

```diff
@@ -957,7 +936,7 @@ def post(self):
             the_search_str=self.request.get('s', None)
             self._make_page(the_user=self.user, the_page=the_page, the_search=the_search_str)
         else:
-            return;
+            self.abort_nonadmin()
```

Comment:
Actually, we have a @superuser_required decorator - should probably do this that way instead. Weirdly, the only place it's in use is in the "forums" module, which is half-baked and not exposed. But it's a neater way to go.